recipe/laravel.php

```diff
@@ -35,6 +35,16 @@
     'storage/logs',
 ]);
 
+set('laravel_version', function () {
+    $result = run('{{bin/php}} {{release_path}}/artisan --version');
+
+    preg_match_all('/version\ (.+?)$/', $result, $matches);
```

Comment:
@barryvdh hmmm. Make sense! In Laravel 5.2, `--version` param come with 'version' string. I will change it. Thanks.